react-components/src/components/Widgets/WindowWidget.tsx

```diff
@@ -102,17 +105,13 @@ export const WindowWidget = ({
             }
           : {}
       }>
-      <Draggable
-        onDrag={handleDrag}
-        position={isMinimized ? { x: 0, y: 0 } : position}
-        handle=".widget-header"
-        disabled={isMinimized}>
+      <Draggable onDrag={handleDrag} position={position} handle=".widget-header">
         <ResizableBox
           width={size.width}
           height={size.height}
           minConstraints={[WIDGET_WINDOW_MIN_WIDTH, WIDGET_INSIDE_WINDOW_MIN_HEIGHT]}
           maxConstraints={[parentContainerElement.clientWidth, parentContainerElement.clientHeight]}
-          resizeHandles={isMinimized ? [] : ['se']}
+          resizeHandles={isMinimized ? [] : ['se', 'ne', 'e', 's']}
```

Comment:
Is there a reason we want these in particular and not e.g. 'n' or `sw`? I also see that dragging the top-right corner upward, it only increases the height of the window downward. But not a critical bug

Reply:
Increase in height downward while dragging top-right corner functionality is coming from `ResizeableBox` component, I could not find a way to fix it, will have a look into it latter.
And I haven't added `n, nw, sw, w` as these are increasing/decreasing height & width in one side that is down and right, it is not supporting top or left ward resize 🙅